### DISCUSSION TITLE
What assurance you can provide that your project will not be moved to commercial use in future?

### DISCUSSION THREAD

**Discussion by ghost** [squidfunk/mkdocs-material#5436] [Q&A] [answered]
Hey @squidfunk ,
What assurance you can provide that your project `mkdocs-material` will not be moved to commercial use or sold to company in future and eventually be something like Docker/GitBook/MongoDB/Redis/Grafana etc. ?

Or should we expect that it could move and as need at the time you may change OSS license to IP as you notice there is wide use of the tool?

**Comment by squidfunk** [comment, marked as answer]
As long as we (I and fellow maintainers like @katharinalisalin) are maintaining Material for MkDocs, it will remain free and licensed under MIT, forever. Insiders is licensed under MIT as well, and we have no plans to change that. If we would think of changing to a non-OSS license, we would've already done that, as Material for MkDocs has a quite large user base. The Insiders program is how we try to make the project sustainable, as it buys us the necessary time to work on the project.

We believe in OSS and sponsorware. [This is also the very reason why we cannot provide access to Insiders for free](https://github.com/squidfunk/mkdocs-material/discussions/5433). If we would do that, we would remove the possibility for us to maintain it, making it necessary to move to other means of financial compensation.
